apps/scoutgame/components/common/Card/BuilderCard/BuilderCard.tsx

```diff
@@ -46,6 +50,9 @@ export function BuilderCard({
   type: 'default' | 'starter_pack';
   showListButton?: boolean;
   scoutInView?: string;
+  actionSlot?: ComponentType<T>;
```

Comment:
I came up with this approach to customizing the action under each card, where we pass in a component and additional props for it. The requirements for adding a button to select a card included:
- be able to pass the week from server component into it
- but also receive the user id of the selected dev card, which is a client component

I used "any" to get typings to work, but if this becomes useful we can figure them out later.